types/plotly.js/index.d.ts

```diff
@@ -1754,6 +1754,25 @@ export interface HoverLabel extends Label {
     namelength: number;
 }
 
+export interface MultiHoverLabel extends Omit<Label, "bgcolor"> {
+    /**
+     * Sets the horizontal alignment of the text content within hover label box.
+     * @default "auto"
+     */
+    align: 'left' | 'right' | 'auto';
+
+    /** Sets the background color of all hover labels on graph or of specific sample. */
+    bgcolor: string | string[];
+
+    /**
+     * Sets the default length (in number of characters) of the trace name
+     * in the hover labels for all traces.
+     * -1 shows the whole name regardless of length.
+     * @default 15
+     */
+    namelength: number;
+}
+
```

Comment:
```suggestion
export interface MultiHoverLabel extends Omit<HoverLabel, "bgcolor"> {
    /** Sets the background color of all hover labels on graph or of specific sample. */
    bgcolor: string | string[];
}

```
does it make sense, to simplify this?
Please also add a test case, Plotly IMO is a hard to grasp for a reviewer.